pkgs/development/python-modules/pyaxmlparser/default.nix

```
@@ -1,16 +1,31 @@
-{ buildPythonPackage, stdenv, lxml, click, fetchPypi }:
+{ buildPythonPackage, stdenv, lxml, click, fetchFromGitHub, pytest, isPy3k }:
 
 buildPythonPackage rec {
   version = "0.3.13";
   pname = "pyaxmlparser";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "1mzdrifnaky57vkmdvg0rgjss55xkxaramci3wpv4h65lmk95988";
+  # the PyPI tarball doesn't ship tests.
+  src = fetchFromGitHub {
+    owner = "appknox";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0jfjhxc6b57npsidknxmhj1x813scg47aaw90ybyr90fpdz5rlwk";
   };
 
+  # runtime fails with `TypeError: AXMLParser object is not an iterator`
+  disabled = !isPy3k;
+
+  postPatch = ''
+    substituteInPlace setup.py --replace "click==6.7" "click==7"
```

To avoid future headaches: 

```suggestion
    substituteInPlace setup.py --replace "click==6.7" "click"
```